theme/scripts/utils/isMobile.ts

```diff
@@ -0,0 +1,14 @@
+const breakpointIsMdUp = 900;
```

Comment:
Just a suggestion: this style of naming convention `${something}Is${Something}` is useful when the return value is a boolean.

So if the was:
```
const breakPointIsMdUp = window.innerWidth >= 900;
```
then this variable name is great.

given that this is really just a constant, a better name would be something like:

`mdBreakpoint`

Reply:
Thanks, always love the tips on naming because I struggle with this!!!

Reply:
Yeah, i think we all do haha